### PR TITLE
DOCS-1337: WAF changes

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:

https://tigera.atlassian.net/browse/DOCS-1337
Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by girishshankaran** [comment]
@electricjesus Can you review this PR?

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 1]
```diff
@@ -55,7 +54,7 @@ With {{prodname}} WAF, you gain visibility into internal east/west traffic at th
 
 **Required**
 
-Configure Felix for syncing WAF policy. Note: this is the same step required for L7 log collection so you may already have this set.
+Configure Felix for syncing WAF policy. If you have already configured L7 logs, you can ignore this step.
```
Comment (electricjesus) [inline calico-enterprise/threat/web-application-firewall.mdx]: perhaps you can link to the L7 logs doc?

```
Configure Felix for syncing WAF policy. If you have already configured [L7 logs](l7-logs-url-here), you can ignore this step.
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 2]
```diff
@@ -65,132 +64,178 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
 
-```yaml
-apiVersion: operator.tigera.io/v1
-kind: ApplicationLayer
-metadata:
-  name: tigera-secure
-spec:
-  webApplicationFirewall: Enabled
+```
+  curl http://<host>//test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user
```
Comment (electricjesus) [inline calico-enterprise/threat/web-application-firewall.mdx:104]: <host> seems ambiguous IMHO..  perhaps something like <service-url>?

**Review by electricjesus — APPROVED** [review]
looking good, just a minor few things

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 3]
```diff
@@ -65,132 +64,178 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
-- [Configure a cluster for WAF](#configure-a-cluster-for-waf)
-- [Add and edit rules](#add-and-edit-rules)
-- [Seeing WAF events in Kibana and Manager UI](#seeing-waf-events-in-kibana-and-manager-ui)
-  - [Kibana](#kibana)
-  - [Manager UI](#manager-ui)
+- [Enable WAF](#enable-waf)
+- [View WAF events](#view-waf-events)
+- [Create global alerts](#create-global-alerts)
+- [Manage WAF](#manage-waf)
+   - [Update rules](#update-rules)
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]: Link is broken. Should be Change WAF rules.

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 4]
```diff
@@ -65,132 +64,178 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
+   - [Disable WAF](#disable-waf)
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]: Link is broken. Disable WAF for a service?

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 3 above)
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]: Change rules
```suggestion
   - [Change WAF rules](#change-waf-rules)
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 5]
```diff
@@ -199,99 +244,57 @@ curl -O https://raw.githubusercontent.com/lsgroup/SmartReverseProxy/master/modse
...
-Now view the WAF logs in Kibana by selecting the `tigera_secure_ee_waf*` index pattern. You should see the relevant WAF assessment from your request recorded:
-
-<img
-  src='/img/calico-enterprise/waf-kibana.png'
-  alt='WAF logs in Kibana'
-  width='600'
-/>
-
-#### Manager UI
+#### Disable WAF featire
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]:
```suggestion
#### Disable WAF feature
```

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 4 above)
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]: ### 
```suggestion
   - [Disable WAF for a service](#disable-waf-for-a-service)
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 6]
```diff
@@ -65,132 +64,178 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
 
-In case of an error, HTTP request will return HTTP 403 Response Code from Envoy to the originating service.
+In case of an error, the HTTP request will return HTTP 403 Response Code from Envoy to the originating service.
+
+ModSecurity provides the following actions that you update in SecAction:
 
 | Action | Description                                                                                                                                                             | Disruptive? |
 | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
 | Block  | Despite the name, this **will not block or drop the request**. ModSecurity will return detection=0 in this case and Calico will **not** log the event in Elasticsearch. | No          |
 | Deny   | Denies HTTP traffic as ModSecurity will return detection=1 and will log the event in Elasticsearch.                                                                     | Yes         |
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]: Deny and Drop are run-on sentences. Hard to parse. Something like:

Drop.     
Denies HTTP traffic. ModSecurity returns detection=1 and logs the event in Elasticsearch.

Deny
Denies HTTP traffic. ModSercurity returns detection=1 and logs the event in Elasticsearch.

**Review by bmckercher123 — APPROVED** [review]
Looks great. Minor issues.

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 4 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 4 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 5 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 4 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 6 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done
